service/src/main/resources/db/changelog.xml

```diff
@@ -10,4 +10,5 @@
     <include file="changesets/20220516_workspace_user_facing_id_required.yaml" relativeToChangelogFile="true"/>
     <include file="changesets/20220617_workspace_activity_log_table.yaml" relativeToChangelogFile="true"/>
     <include file="changesets/20220721_actor_email_and_actor_subject_id_columns.yaml" relativeToChangelogFile="true"/>
+    <include file="changesets/20220802_default_value_to_actor_email_and_actor_subject_id_columns.yaml" relativeToChangelogFile="true"/>
```

Comment:
I don't think this will work. I think you need to edit `20220721_actor_email_and_actor_subject_id_columns.yaml` and provide the defaults there.
In its current form it cannot be applied by liquibase if there are any existing entries in the activity log table.

Reply:
oh i thought i'm not allowed to change liquidbase file once it's in.

Reply:
Yes, that is the general practice. But when the liquibase file doesn't work, then we need to fix it. Since it has never been deployed anywhere except local testing, it is fine to revise it.